### ISSUE TITLE
Cannot change cluster selection strategy for class

### ISSUE THREAD

**Issue by mxns** [orientechnologies/orientdb#6897]
## OrientDB Version, operating system, or hardware.
- v2.2.12-SNAPSHOT

## Operating System
- [x ] MacOSX

## Expected behavior and actual behavior

Expected: according to documentation, the _cluster selection strategy_ for a class in distributed mode should be set to `local`. 

Actual: _cluster selection strategy_ is set to _round-robin_, and cannot be changed. This leads to error when doing insert, as every _X:th_ insert will fail since it is targeted at a cluster that the current node does not own.

## Steps to reproduce the problem

- Download v2.2.12, unzip
- Run _bin/dserver.sh_
- Login with console application:  _connect remote:localhost/GratefulDeadConcerts root password_
- Try changing cluster selection strategy for class _sung\_by_

```
orientdb {db=GratefulDeadConcerts}> info class sung_by

CLASS 'sung_by'

Records..............: 501
Super classes........: [E]
Default cluster......: sung_by (id=41)
Supported clusters...: sung_by(41), sung_by_1(42), sung_by_2(43), sung_by_3(44), sung_by_4(45), sung_by_5(46), sung_by_6(47), sung_by_7(48)
Cluster selection....: round-robin
Oversize.............: 0.0
orientdb {db=GratefulDeadConcerts}> alter class sung_by clusterselection local

Class updated successfully.

orientdb {db=GratefulDeadConcerts}> info class sung_by

CLASS 'sung_by'

Records..............: 501
Super classes........: [E]
Default cluster......: sung_by (id=41)
Supported clusters...: sung_by(41), sung_by_1(42), sung_by_2(43), sung_by_3(44), sung_by_4(45), sung_by_5(46), sung_by_6(47), sung_by_7(48)
Cluster selection....: round-robin
Oversize.............: 0.0
```

**Comment by mxns** [comment]
Additional info: inserts work fine until a node is restarted: 
[http://stackoverflow.com/questions/40506577/inserts-javaapi-fail-after-restarting-node-in-distributed-orientdb-cluster](http://stackoverflow.com/questions/40506577/inserts-javaapi-fail-after-restarting-node-in-distributed-orientdb-cluster)

**Comment by lvca** [comment]
@mxns we have the same issue reported elsewhere. I'm working on it.

**Comment by mxns** [comment]
Ok, thanks for the update @lvca. And your work :) If you wanna point me to a relevant github issue that I can follow and close this one, that's fine.

**Comment by lvca** [comment]
@mxns Thanks for reporting it. It's fixed in 2.2.x branch. Please could you check it? NP, I'm going to take care of the related issues.

**Comment by mxns** [comment]
Yup, the issue that I described in [this stackoverflow entry](http://stackoverflow.com/questions/40506577/inserts-javaapi-fail-after-restarting-node-in-distributed-orientdb-cluster) is fixed on 2.2.x, nice work @lvca. 

The cluster selection strategy is still round-robin though, so it seems I misunderstood the underlying problem? Anyway, thanks!

**Comment by lvca** [comment]
If you dump the name is round-robin, but it's wrapped in the Local strategy wrapper.
